Dockerfile

```
@@ -0,0 +1,28 @@
+
+# Set working directory
+WORKDIR /app
+
+# Copy the project files
+COPY . .
+
+# Remove any existing build directory and CMake cache
+RUN rm -rf build CMakeCache.txt
+
+# Create build directory and build the project
+RUN mkdir build && \
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)